hoomd/mpcd/stream.py

```diff
@@ -605,7 +608,69 @@ def __init__(self, R, boundary="no_slip", period=1):
                                  hoomd.context.current.system.getCurrentTimeStep(),
                                  self.period,
                                  0,
-                                 _mpcd.SphereGeometry(R,bc))
+                                 _mpcd.SphereGeometry(R, bc))
+
+    def set_filler(self, density, kT, seed, type='A'):
+        r""" Add virtual particles outside the spherical confinement
+
+        Args:
+            density (float): Density of virtual particles.
+            kT (float): Temperature of virtual particles.
+            seed (int): Seed to pseudo-random number generator for virtual particles.
+            type (str): Type of the MPCD particles to fill with.
+
+        The virtual particle filler draws particles in *all space outside* the spherical wall since
+        it would be very tricky to determine the number of virtual particles to add close to the wall
+        (individual ranks) due to the curvature of the geometry. The particle positions are drawn
+        from an uniform distribution and the velocities from distribution consistent with *kT* and
+        with the given *density*. Typically, the virtual particle density and temperature are set to
+        the same conditions as the solvent.
+
+        The virtual particles will act as a weak thermostat on the fluid, and so energy is no longer
+        conserved. Momentum will also be sunk into the walls.
+
+        *NOTE*: Simulation box-size should be chosen carefully. The box should be just large enough to
+        hold the sphere+padding layer of mpcd cells (best case single padding layer). Performance of the
+        filler degrades with increasing box-size, since we are using rejection sampling method.
```

Comment:
```suggestion
        Note:

            The simulation box size should be chosen carefully. The box should be just large enough to
            hold the sphere+padding layer of cells (best case single padding layer). Performance of the
            filler degrades with increasing box size, since we are using rejection sampling method.
```